### DISCUSSION TITLE
How to deploy minio in Linux production environment and use two domain names for nginx external and internal access proxy? And start the account and password to log in to the management interface and report 401

### DISCUSSION THREAD

**Discussion by wdjmirrors** [minio/minio#20538] [Q&A]
minio如何在Linux生产环境进行部署，并用两个域名进行nginx对外对内访问代理？且启动账号密码登录管理端界面报401

**Comment by wdjmirrors** [comment]
我的启动脚本配置：
export MINIO_ROOT_USER=root 
export MINIO_ROOT_PASSWORD=xxxxxxxx
export MINIO_SERVER_URL=https://xxxxx.cn
export MINIO_BROWSER_REDIRECT_URL=https://xxxxxx.cn/platform/files/

./minio server --address 0.0.0.0:9000 --console-address 0.0.0.0:9111 /usr/local/minio/data > /usr/local/minio/minio.log 2>&1 &

**Reply by wdjmirrors** [reply]
My startup script start-minio.sh configured:

export MINIO_ROOT_USER=root
export MINIO_ROOT_PASSWORD=xxxxxxxx
export MINIO_SERVER_URL=https://xxxxx.cn/
export MINIO_BROWSER_REDIRECT_URL=https://xxxxxx.cn/platform/files/

./minio server --address 0.0.0.0：9000 --console-address 0.0.0.0：9111 /usr/local/minio/data > /usr/local/minio/minio.log 2>&1 & &

**Comment by klauspost** [comment]
English Please.
